restaurant/models/restaurant.py

```diff
@@ -0,0 +1,65 @@
+from django.core.exceptions import ValidationError
+from django.core.validators import MinValueValidator
+
+from django.db import models
+
+
+class RestaurantType(models.Model):
+
+    """This class is using for creating table with types of Restaurants"""
+
+    rest_type = models.CharField(max_length=256, blank=False)
+    is_deleted = models.BooleanField(default=False)
+
+    class Meta(object):
+
+        """Correct displaying of restaurant type in restaurant's list"""
+
+        verbose_name=u"Restaurant type"
+
+    rest_type = models.CharField(max_length=256, blank=False)
+
+    def __unicode__(self):
+        return u"%s" % (self.rest_type)
+
+class Restaurant(models.Model):
+
+    """This class describe model of restaurant object,
```

Comment:
Please, fix this according to PEP 257.

> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".
[PEP 257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings)